webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7313,7 +7355,44 @@ var CURRENT_GROUP_DATA={};
                 };
                 return conversationDetail;
             };
-
+            _this.getBotDetails = function () {
+                window.Applozic.ALApiService.ajax({
+                    url: MCK_BOT_API + "/bot/" + MCK_APP_ID,
+                    type: 'get',
+                    global: false,
+                    success: function (data) {
+                    var flag = false;
+                    console.log(data);
+                    for (var key in data)
+                    {
+                        if (data.hasOwnProperty(key)) {
+                            if(data[key].name == CURRENT_GROUP_DATA.conversationAssignee){
+                                //console.log(data[key].name);
+                                if(data[key].aiPlatform == 'dialogflow')
+                                {
+                                    if(!data[key].autoHumanHandoff){
+                                        CHAR_CHECK = true;
+                                        document.getElementsByClassName('mck-char-warning')[0].style.backgroundColor = WIDGET_SETTINGS.primaryColor;
+                                        break;
+                                    }else{
+                                        CHAR_CHECK = false;
+                                    }
+                                }else{
+                                    CHAR_CHECK = false;
+                                }
+                            }else{
+                                CHAR_CHECK = false;
+                            }
+                        }
+                    }
+                    },
+                    error: function () {
+                            w.console.log('Unable to load bot info. Please reload page.');
```

Comment:
@hardikparashar whats the difference between w.console.log vs console.log, both are getting used.

Reply:
removed

Reply:
changed in new commit